modes/parkour/interface.lua

```diff
@@ -587,12 +588,23 @@ onEvent("ChatCommand", function(player, msg)
 
 		save_update = os.time() + 60000 * 3 -- 3 minutes
 		translatedChatMessage("action_within_minute", player)
+			
+	elseif cmd == "map" then
+		
+		newMap()
+			
+	elseif cmd == "a" and #args > 0 then
```

Comment:
I don't like the command name. You can use the pointer variable which is meant to have the exact length of args.

Reply:
got rid of that